src/fortran_routing/Reservoir_Binding/Reservoirs/makefile

```diff
@@ -1,4 +1,4 @@
-NETCDFINC := $(shell nc-config --fflags)
+NETCDFINC := -I$(NETCDFINC) $(shell nc-config --fflags)
```

Comment:
@hellkite500 @jameshalgren Would the be the right update to make to the makefile permanently?

Reply:
Yeah, it just prepends the ENV variable `NETCDFINC` if it exists.